src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf

```diff
@@ -468,7 +468,7 @@
             </trans-unit>
             <trans-unit id="121">
                 <source>This value is not a valid Twig template.</source>
-                <target state="needs-review-translation">Deze waarde is geen geldige Twig-template.</target>
+                <target>Deze waarde is geen geldige Twig template.</target>
```

Comment:
as you removed the `-`, can you also double check line 455, if the `-` is needed there? Thanks

Reply:
> as you removed the `-`, can you also double check line 455, if the `-` is needed there? Thanks

Thanks for your reply, the `-` is required in ISO 8601-format ([reference](https://onzetaal.nl/taalloket/afkorting-in-samenstelling) in Dutch).

For `Twig Template` both with and without `-` seems valid, it depends on (personal) preferences. However, when compared to for example Word-bestand, the `-` seems more common (as Microsoft mentions it there Dutch documentation with `-`). I thereby updated the PR  with `-`.